src/styles/main.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  max-width: calc((#{$day-size} * 7 + (6 * #{$gap})) + #{$padding} * 2);
```

Comment:
The usage of arithmetic operations inside `calc` function should follow the format with spaces around the `+` and `-` operators for better readability. Consider revising to `calc((#{$day-size} * 7) + (6 * #{$gap}) + (#{$padding} * 2))`.